_data/docs.yml

```diff
@@ -90,6 +90,8 @@
       keywords : ["fridge monitoring","fridge"]
     - title: Leak Detector Installation Guide
       keywords : ["leak","water"]
+    - title: Feedback Station Installation
+      keywords : ["feedback station","facilities"]
 
```

Comment:
It might be interesting to put the feedback before the "leak" so we keep an alphabetical order for specific installs

Reply:
IDK how to change the content file name